### PR TITLE
issue-17 : persist port data in db

### PR DESCRIPTION
**Pull request by bharath-b-hpe**

### CODE REVIEW

**Review by jeevan-kamkar — APPROVED** [review]

**Review by a-ajith — APPROVED** [review]

File: caphandler/addresspools.go [hunk 1]
```diff
@@ -37,19 +36,17 @@ func GetAddressPoolCollection(ctx iris.Context) {
...
-
-	var members = []*model.Link{}
-	for addressPoolOID, addressPooldData := range capdata.AddressPoolDataStore {
-		if addressPooldData.FabricID == fabricID {
-			members = append(members, &model.Link{
-				Oid: addressPoolOID,
-			})
-		}
+	for addressPoolOID := range addressPools {
+		members = append(members, &model.Link{
```
Comment (Bharath-KKB) [inline caphandler/addresspools.go:47]: why did we remove the check for fabricID?

File: caphandler/endpoints.go [hunk 2]
```diff
@@ -37,20 +37,21 @@ import (
...
-	var members = []*model.Link{}
 
-	for endpointID, endpointData := range capdata.EndpointDataStore {
-		if endpointData.FabricID == fabricID {
-			members = append(members, &model.Link{
-				Oid: endpointID,
-			})
-		}
+	var members = []*model.Link{}
+	for endpointID := range endpointData {
```
Comment (Bharath-KKB) [inline caphandler/endpoints.go:49]: end point is not attached to a fabric anymore? But it should have a parent.

**Review by Bharath-KKB — CHANGES_REQUESTED** [review]
team to review again. and fix my comments.

File: caphandler/addresspools.go (hunk 1 above)
Comment (bharath-b-hpe) [reply caphandler/addresspools.go:47]: Key set name contains the fabric ID using which we fetch the address pool data, hence additional DB call to check the fabric ID was removed.

File: caphandler/endpoints.go (hunk 2 above)
Comment (bharath-b-hpe) [reply caphandler/endpoints.go:49]: Key set name used for storing address pool, zones and endpoints all have the fabric ID as the key, and all queries to fetch these data are using fabric ID.

File: caphandler/addresspools.go (hunk 1 above)
Comment (Bharath-KKB) [reply caphandler/addresspools.go:47]: will that handle multiple fabric IDs ?

File: caphandler/endpoints.go (hunk 2 above)
Comment (Bharath-KKB) [reply caphandler/endpoints.go:49]: as above

File: caphandler/addresspools.go (hunk 1 above)
Comment (bharath-b-hpe) [reply caphandler/addresspools.go:47]: yes, multiple unique fabric IDs is supported

File: caphandler/endpoints.go (hunk 2 above)
Comment (bharath-b-hpe) [reply caphandler/endpoints.go:49]: yes, multiple unique fabric IDs is supported

**Pull request merged**